prime_factor/PrimeFactor.java

```diff
@@ -0,0 +1,28 @@
+import java.util.*;
+
+public class PrimeFactor {
+    public static ArrayList<Integer> primeFactor(int n)
+    {
```

Comment:
Opening bracket should be on same line. Please add this `{` after `)` on line 4.